.github/workflows/ci-nix.yml

```diff
@@ -305,6 +305,14 @@ jobs:
           nix_tag=${{ env.gatewayd_container_tag }} && hub_tag="fedimint/gatewayd:master" && docker tag "$nix_tag" "$hub_tag" && docker push "$hub_tag"
           nix_tag=${{ env.gateway-cli_container_tag }} && hub_tag="fedimint/gateway-cli:master" && docker tag "$nix_tag" "$hub_tag" && docker push "$hub_tag"
 
+      - name: Publish tagged release
+        if: github.ref_type == 'tag'
```

Comment:
If I understand correctly, this _shouldn't_ work because the workflow runs on branch push, not tag push.

Reply:
Added to push:
```yaml
    tags:
      - 'v*'
```